### ISSUE TITLE
Increase Meta Description Custom Text to 320 Characters [Feature Request]

### ISSUE THREAD

**Issue by bfopma** [khalwat/seomatic#304]
Per [https://moz.com/blog/googles-longer-snippets](https://moz.com/blog/googles-longer-snippets) Google is now allowing meta descriptions to be "not greater than 320 characters." Can the SEO Description custom text option be modified to allow up to 320 characters?

Thank you!

**Comment by gutestun** [comment]
Agreed. An indicator when the description is between 230 and 320 characters would be optimal.

**Comment by khalwat** [comment]
You can do this via the `config.php`:

https://github.com/nystudio107/seomatic/blob/master/config.php#L26

**Comment by khalwat** [comment]
> Just wanted to suggest a small feature change on SEOmatic now when Google is showing more than 160 characters on the description. 
> In my config/seomatic.php I've made the changes:
> 
> "maxDescriptionLength" => 360,
> 
> ....but the fieldtype still shows 160 characters countdown because that number is hardcoded in /templates/field.php. 
> If that variables also could be "maxDescriptionLength" it would be great. Now I had to change it manually in that file.

**Comment by treemonaco** [comment]
I have set the following...

```
/**
 * The maximum number of characters allow for the seoDescription.  It's HIGHLY recommend that
 * you keep this set to 160 characters.
 */
    "maxDescriptionLength" => 320,

/**
 * Controls whether SEOmatic will truncate the descrption tags maxDescriptionLength characters.
 * It is HIGHLY recommended that you leave this on, as search engines do not want
 * description tags to be long.
 */
    "truncateDescriptionTags" => false,
```

as well as increased the length in the templates/field file and the description is still truncating at 160 when viewed in the <head>.

Is there something I'm missing or does this simply not work yet?
